lib.sh

```diff
@@ -770,9 +770,9 @@ else
     if ! netplan apply
     then
         systemctl restart systemd-networkd > /dev/null
+        countdown 'Waiting for network to restart...' 3
     fi
     # Check the connention
-    countdown 'Waiting for network to restart...' 3
```

Comment:
Why this change?

Reply:
Because I though this would be only needed if we actually restart the network in the if-statement above?
Or does netplan apply actually restart the network so that we have to wait for this 3 seconds?

Reply:
> Or does netplan apply actually restart the network so that we have to wait for this 3 seconds?

It should check if the network is OK even after netplan apply, so yes, it's needed where it was from the beginning. :)

Reply:
Hm, I don't get it. I just moved that line two lines higher, I didn't remove it...

Reply:
Yeah, and now you don't get if network is online if `netplan apply` exits with 0, which is wrong - because it should check after `netplan apply` as well. :)

Reply:
But the countdown function doesn't check if the network is online, does it?

Reply:
No, but right now it doesn't wait for `netplan apply` either, which it should.

Reply:
I've reverted that change.